{
    "data_folder_path": "data/CVE-2018-16876",
    "pull_requests_urls": [
        "https://github.com/ansible/ansible/pull/49569"
    ],
    "cve_name": "CVE-2018-16876",
    "repo": "ansible/ansible",
    "warnings": {
        "cve": "CVE-2018-16876",
        "warnings": [
            {
                "type": "NOT_MODIFIED",
                "file_name": "changelogs/fragments/avoid_ssh_retry_discolsures.yml",
                "message": "not a modified file (status: FileStatus.ADDED)\n"
            }
        ]
    },
    "files_extension_stats": {
        ".yml": 1,
        ".py": 1
    },
    "file_number": 2,
    "removed_files_number": 0,
    "added_files_number": 0,
    "nb_lines_affected": 17,
    "private_key_files": [],
    "languages": [
        "Python"
    ]
}